app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -340,9 +340,17 @@ public void insertMultipleChoiceCard(MultipleChoiceCard mcCard, SQLiteDatabase d
     }
 
     public void deleteMultipleChoiceCard(MultipleChoiceCard mcCard) {
-        db.delete(MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,
-                "id" + "=" + mcCard.getId(),
-                null);
+        Cursor res = db.rawQuery("SELECT * FROM " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,null);
+        int value = res.getColumnIndex("id");
+        if (value != -1) {
+            db.delete(MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,
```

Comment:
Why are we checking if the value is not equal to -1? Why -1 i don't get it..

Reply:
When a column doesn't exist then the result will be -1. So basically if the column exists I'm conducting my delete statement. I need to do this because my unit tests look for the column "id" while the app itself uses the column "_id". I dont know why...but I have to make a check.